pyo3-macros-backend/src/introspection.rs

```diff
@@ -333,3 +375,59 @@ fn unique_element_id() -> u64 {
         .hash(&mut hasher); // If there are multiple elements in the same call site
     hasher.finish()
 }
+
+/// Hack function that changes explicit lifetime parameters like 'py into '_ to be able to use the type outside of its position
+fn remove_bound_lifetimes(t: Type) -> Type {
```

Comment:
You can probably remove this and use `t.elide_lifetimes()` from our `TypeExt` trait in `utils.rs`.

Reply:
Thank you! I missed it. Done!